### DISCUSSION TITLE
Can you create an alternative sign up experience?

### DISCUSSION THREAD

**Discussion by Bovelett** [community/community#49454] [Accessibility] [answered]
### Select Topic Area

Product Feedback

### Body

Dear GitHub Team,

In the process of writing an article about how to post bug reports and feature requests for first time users on GitHub, I wanted to create a video about how the sign-up process works. My article is targeted at the hundreds of thousands of active Elementor users. Elementor has a very active place in the realm of GitHub.

As a user who often depends on keyboard navigation, I am generally happy with the experience on GitHub. It's been quite a while ago since I signed up, in 2011. Given the smooth experience I normally have when navigating your site, I was expecting a smooth and clean process.

And yes, in a way it is. Fast and easy. For people who do are not hindered by any accessibility issues. As an accessibility advocate, web designer, but also being part of the accessibility target group (ADHD, dyslexia, cognitive issues) I was literally blown away by the new sign up experience. Both in a good sense but also in a less good sense.

### The sign-up pages and animations are stunning and beautiful!
And really, I get it why this is made like that. But for some (like me and thousands of people like me) the moving elements are overwhelming to an extent that it literally causes nausea or migraines.

You'd be surprised at the high number of people who, consciously and unconsciously, feel discouraged to sign up because of that. I am quite certain that this was never your intent.

### Suggestion: offer a visually simplified sign-up
In all honesty, I'm just throwing this over the fence, without a clear vision of "how". But not saying anything is not an option. My question is if you would consider giving new users an option to visit a visually simplified sign-up. I have seen another thread where you suggested people turn on reduced motion settings. But that is assuming all users affected by this are aware of this option. Many are not.

### Let me know if I can help!

Cheers from Germany,

Anne

**Comment by cehfisher** [comment]
Hello @Bovelett - thank you for providing this feedback! We will send this feature request for a alternative sign-up experience to our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. While we can’t guarantee immediate action on every piece of feedback, rest assured that we are attentive to your suggestions and always striving to enhance the GitHub experience. Thanks again!

**Comment by cehfisher** [comment, marked as answer]
Hi @Bovelett - Just a quick follow-up that we're actively working on this internally, but we don't have a specific timeline or further details to share at this time. Thank you for your patience, and we'll keep you updated as we make progress.

**Reply by Bovelett** [reply]
Hi @cehfisher, thank you for letting me know!

**Comment by cehfisher** [comment]
Hey @Bovelett – a new sign-up experience recently launched. It streamlines the process, keeps accessibility in mind, and stays true to the brand. We'd love for you to try it out and share your thoughts!

**Comment by Bovelett** [comment]
Hey Carie @cehfisher - This is so brilliant! ❤️ I was in happy tears when I saw this, such a work of love. By sheer coincidence, I wrote an [article about creating animation with accessibility in mind](https://annebovelett.eu/gsap-and-accessibility-yes-you-can-have-both/) last week. This is such a great example how amazing design and accessibility go hand in hand. Lots of love to you and all who worked so hard to make this happen. I will write an article about this. Thank you for keeping me updated over time. I really appreciate it.

**Reply by cehfisher** [reply]
Thank you for retesting the new sign-up experience and for writing such an insightful article — I’ll be sharing both with the team ❤️  

I’m closing this discussion as resolved, but feel free to reopen it or start a new thread if you have more feedback. Thanks again!

**Reply by bgonzalez** [reply]
Yay! Thanks for Not giving up @Bovelett  and thanks to you @cehfisher abs The Team for making it Happen with so much love showing that such a delightful UX can also be accessible! 🥹🤟
